src/js/app/model/asset.js

```diff
@@ -2,6 +2,7 @@
 
 import Backbone from 'backbone';
 import THREE from 'three';
+const placeholderUrl = require("file!../../../../img/placeholder.jpg");
```

Comment:
@lirsacc with move to Webpack we need to let Webpack know about all the assets we use. The only case I had to change was this - we require(``) and use the path relative from the development environment, not build env. Webpack file loader will replace this with the build env link to the file.